src/components/Header/MenuMobile.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import cn from "classnames"
+import NavigationLinks from "../NavigationLinks/NavigationLinks"
+import Container from "../Container/Container"
+import LinkLaunchApp from "../Link/LinkLaunchApp"
+import s from "./MenuMobile.module.css"
+
+interface IMenuMobileProps {
+  className?: string
+}
+
+const MenuMobile: React.FC<IMenuMobileProps> = ({ className }) => {
+  return (
```

Comment:
You can use an implicit return

Reply:
actually, I just use the template here) so it's happens every time)